boot/nxboot/Kconfig

```diff
@@ -92,6 +103,10 @@ config NXBOOT_PREVENT_DOWNGRADE
 		WARNING: NXboot currently implements preferences only for
 		MAJOR.MINOR.PATCH and ignores prerelease.
 
+config NXBOOT_PRINTF_PROGRESS
+	bool "Enable progress messages to be sent to STDOUT"
+	default n
```

Comment:
I think a progress percentage (or even a progress bar), is something expected to exist by default (y), maybe the option should be here to disable it.

Reply:
> I think a progress percentage (or even a progress bar), is something expected to exist by default (y), maybe the option should be here to disable it.

I was trying to make sure the behaviour remained the same for existing builds with no additional KiB used unless you want and choose this option?

Reply:
I don't have a strong opinion on this. I personally prefer less verbose output, but it may be useful to users not familiar with the bootloader. So maybe we could enable it by default.

I would definitely add `---help---` with the description how much size does it take.

Reply:
Added